runtime/compiler/runtime/IProfiler.cpp

```diff
@@ -4156,8 +4155,10 @@ UDATA TR_IProfiler::parseBuffer(J9VMThread * vmThread, const U_8* dataStart, UDA
             callee = *(J9Method**)cursor;
             cursor += sizeof(J9Method *);
 
-            if (callee && javaVM->initialMethods.initialVirtualMethod != callee &&
-               !fanInDisabled)
+            if (callee
+               && (javaVM->initialMethods.initialVirtualMethod != callee)
+               && (javaVM->initialMethods.invokePrivateMethod != callee)
```

Comment:
I strongly object to this inline code - if making a proper API is too hard, I suggest a macro taking the symbol address and javaVM.